azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -448,6 +449,14 @@ internal open class CallCompositeActivity : AppCompatActivity() {
         }
     }
 
+    private fun setNavigationBarColor() {
```

Comment:
is it coming for develop or as part of your code change? I think this was causing issue that we discussed?